src/foam/dao/MDAO.java

```diff
@@ -74,8 +74,40 @@ public void detach() {
   protected Object   writeLock_ = new Object();
   protected Set      unindexed_ = new HashSet();
 
-  public final static String GET_MDAO_CMD = "GET_MDAO_CMD"; 
-  
+  /**
+   * DAO Command issued on a DAO stack to return the MDAO.
+   * Used by Medusa to directly update the MDAO.
+   */
+  public final static String GET_MDAO_CMD = "GET_MDAO_CMD";
```

Comment:
If you're grabbing the MDAO directly then do you need the other commands?

Reply:
Although, to be clear, I think it's a better idea to have the other commands than direct access to the MDAO, since the CMDs allow for decoration. So my question is really: do you need both?